### PR TITLE
Adding output_metric properties to sensu_check

### LINKED ISSUE #27: Support output_metric_format and output_metric_handlers
### Chef Version

13.7.16

### Expected Behavior

the sensu_check resource should support the output_metric_format and output_metric_handlers fields

```ruby
sensu_check 'dummy_ping' do
  command 'dummy_ping_check'
  interval 15
  subscriptions %w(
    all
  )
  handlers %w(
    influxdb
  )
  runtime_assets %w(
    dummy_checks
  )
  publish true
  action :create
  output_metric_format 'nagios_perfdata'
  output_metric_handlers %w(
    influxdb
  )
end

```
### Actual Behavior

```ruby
  output_metric_format 'nagios_perfdata'
  output_metric_handlers %w(
    influxdb
  )
```
have no definition

### PR DESCRIPTION
**Pull request by jkerry**
## Pull Request Checklist

resolves #27 by adding output_metric_format and output_metric_handlers

#### General

- [x] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [x] Update README with any necessary configuration snippets

- [x] Cookstyle (rubocop) passes

- [x] Foodcritic passes

- [x] Rspec (unit tests) passes

- [x] Inspec (integration tests) passes

#### New Features

- [ ] Added a [Testing Artifact](https://github.com/sensu-plugins/community/blob/master/PULL_REQUEST_PROCESS.md#7-testing-artifacts) as either an automated test or a manual artifact on the PR.

- [x] Adedd documentation for it to the `README.md`

### CODE REVIEW

**Comment by majormoses** [comment]
We do have some testing but I dont think all of them are tested: https://github.com/sensu/sensu-go-chef/blob/f286c0e133fb5241637908bcbce4649d97e569a6/test/integration/default/default_spec.rb#L45-L59 and https://github.com/sensu/sensu-go-chef/blob/f286c0e133fb5241637908bcbce4649d97e569a6/test/cookbooks/sensu_test/recipes/default.rb#L9-L22

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 - Added skel files from Chef Partners cookbook generator (@thomasriley)
 
 ### Changed
+- Adding `output_metric` settings to the `sensu_check` resource
```
Comment (majormoses) [inline CHANGELOG.md]: This should go under `### Added`

**Comment by jkerry** [comment]
thanks @majormoses . I moved the changelog entry.  I don't think additional inspec tests are justified for such a minor addition (imo) since I'm not sure what output format /bin/true is in.

**Comment by majormoses** [comment]
Thanks for the contribution!

**Pull request merged**